src/amo/api/index.js

```diff
@@ -13,10 +50,16 @@ export function submitReview({
   body,
   reviewId,
   ...apiCallParams
-}) {
+}: SubmitReviewParams): Promise<ApiReviewType> {
   return new Promise(
     (resolve) => {
-      const data = { rating, version: versionId, body, title };
+      const data = {
```

Comment:
Actually, naming it `review` instead achieves the same conciseness that `data` provided.